_includes/backers.md

```diff
@@ -0,0 +1,6 @@
+## Backers
+
+Find Mocha helpful?  Become a [backer](https://opencollective.com/mochajs#support) and support Mocha with a monthly donation. 
```

Comment:
this is a habit I picked up in high school.  think it was APA style, or something.  heated debate out there regarding this, actually.   

but: the only way to get two consecutive spaces in HTML is to use `&nbsp;`, so it doesn't really matter here.